SteamBot/Bot.cs

```diff
@@ -1083,6 +1084,55 @@ public void AcceptAllMobileTradeConfirmations()
                 }
             }                        
         }
+        
+        /// <summary>
+        /// Get duration of escrow in days. Call this before sending a trade offer.
+        /// Credit to: https://www.reddit.com/r/SteamBot/comments/3w8j7c/code_getescrowduration_for_c/
+        /// </summary>
+        /// <param name="partnerId">Partner ID of user you want to send a trade offer to</param>
+        /// <param name="token">User's trade token. Can be an empty string if user is on bot's friends list.</param>
+        /// <returns>TradeOfferEscrowDuration</returns>
+        public TradeOfferEscrowDuration GetEscrowPartnerDuration(int partnerId, string token)
```

Comment:
Probably better to use `SteamID` as the first argument. Can be easily "converted" to AccountID (= PartnerID).